lib/ui/plugins.dart

```diff
@@ -4,28 +4,61 @@
 
 part of dart.ui;
 
+/// Functionality for Flutter plugin authors.
 abstract class PluginUtilities {
-  static Function getClosureByName({String name, Uri libraryUri, String className}) {
+  /// Get a closure instance for a specific static or top-level function given
```

Comment:
This should warn that it will fail to find the closure and return null if the closure is not otherwise used in the program. That is, it may get tree-shaken away in an AOT build.

Reply:
Done.